cobc/codegen.c

```diff
@@ -121,6 +121,12 @@ struct call_list {
 	const char		*call_name;
 };
 
+struct java_call_list {
+    struct java_call_list *next;
+    const char *call_name;
+    cob_java_handle *handle;
```

Comment:
Why do you need the handle here? This should be in the generated code, not in the compiler itself.

Reply:
It is due to this, I am assuming this function must return a void:

`static cob_java_handle* 
lookup_java_call(const char *p, struct cb_call s)`

However:

`cob_java_handle* static_java_method = lookup_java_call(s + 6, p->argv[0]);`

we had this inside the string comparison, so I wanted to confirm this

Reply:
No the call to `cob_resolve_handle` should just be printed via calls to the `output_*` functions in `codegen.c`. Similarly to `cob_resolve_cobol` which is output a few lines below.